isis/src/base/objs/TProjection/TProjection.cpp

```diff
@@ -1564,9 +1564,10 @@ namespace Isis {
       }
       if (!m_good) {
         minBorderX = Null;
-        minBorderY = minBorderX;
-        minBorderX = minBorderX;
-        minBorderY = minBorderX;
+        minBorderY = Null;
+        // minBorderY = minBorderX;
```

Comment:
i would go ahead and just remove these commented-out lines

Reply:
Oops forgot to remove those. Thank you!